src/main/java/lotto/domain/LottoNumber.java

```diff
@@ -0,0 +1,44 @@
+package lotto.domain;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import javax.swing.text.html.Option;
+
+public class LottoNumber {
+	private List<Integer> lottoNumber;
+
+	public LottoNumber() {
+	}
+
+	public LottoNumber(List<Integer> lottoNumber) {
+		this.lottoNumber = lottoNumber;
+	}
+
+	public List<Integer> getLottoNumber() {
+		return lottoNumber;
+	}
+
+	public void generate(NumberGenerator numberGenerator) {
+		this.lottoNumber = numberGenerator.generateNumbers();
+	}
+
+	private int countSameNumber(LottoNumber winLottoNumber) {
+		return (int) lottoNumber.stream()
+			.filter(ln -> winLottoNumber.getLottoNumber().contains(ln))
```

Comment:
```suggestion
			.filter(ln -> winLottoNumber.lottoNumber.contains(ln))
```
같은 타입이면 getter를 쓸 필요가 없겠네요